doc-content/shared-kie-docs/src/main/asciidoc/Workbench/General/customizing-project-preferences-proc.adoc

```diff
@@ -0,0 +1,91 @@
+[id='customizing-project-preferences-proc_{CONTEXT}']
+= Customizing project preferences
+
+A project stores assets and is part of a space. A space can hold multiple projects.
+
+For example, an organization has many departments, such as HR, Payroll, Engineering, R&D, and so on. Each department maps to a space and every department can have its own projects.
+
+You can create a new project from scratch or clone projects from an existing Git repository.
+
+.Procedure
+. In {CENTRAL}, select the *Admin* icon in the upper-right corner of the screen and select *Projects*.
++
+The *Projects* page opens.
```

Comment:
This can be confusing as in the Business central menu there's "Projects". I am not sure about the "Projects page" words together. Would it be possible to rephrase this? Maybe use "settings" instead of page?

Reply:
This is fluff. I have removed it from the content. I think it is not necessary to mention that it opens. The user directly gets access to the **Project Preferences** panel.